src/test/rgw/test_rgw_manifest.cc

```diff
@@ -20,14 +20,6 @@
 #define GTEST
 #ifdef GTEST
 #include <gtest/gtest.h>
-#else
```

Comment:
i'd suggest remove
```c++
#define GTEST
#ifdef GTEST
```
and the closing
```c++
#endif
```
as well.

Reply:
done